### ISSUE TITLE
Add testID and accessibilityLabel props

### ISSUE THREAD

**Issue by a-en** [testshallpass/react-native-dropdownalert#143]
Hi!
Could you review possibility to expose `testID` ([link](https://facebook.github.io/react-native/docs/view#testid)) property? We use dropdown alert for showing various message and it would be really helpful to identify it for testing purposes. Thanks!

**Comment by testshallpass** [comment]
@a-en Hi, where would the `testID` be exposed at? highest level component?

**Comment by a-en** [comment]
Hey!
It would be cool to have it for `TouchableOpacity` so it potentially gives us a method to interact with the alert (if needed). Also, I have to mention that testID doesn't work for Android, so we use `accessibilityLabel` for this platform. Do you think it can be done in that way?

```
<TouchableOpacity
           testID={‘dropdown-alert’} //iOS
           accessibilityLabel={‘dropdown-alert’} //Android
 >
```

**Comment by testshallpass** [comment]
**Acceptance criteria**
* Expose testID as prop.
* Expose accessibilityLabel as prop.
* Document them in README.

**Comment by testshallpass** [comment]
Added to version 3.8.0
